pkg/installmanager/installmanager.go

```diff
@@ -626,8 +635,10 @@ func (m *InstallManager) provisionCluster() error {
 
 	if err := m.runOpenShiftInstallCommand("create", "cluster"); err != nil {
 		if m.isBootstrapComplete() {
```

Comment:
Add a check here for if `m.waitForInstallCompleteExecutions` is greater than zero. If it is not, then there is no need to run `m.isBootstrapComplete()`. That will keep the logs a bit cleaner.